File: src/struct_frame/boilerplate/ts/frame_headers/header_none.ts
// Header None - No start bytes (TypeScript)
// Format: No start bytes, payload begins immediately

import { HeaderType, HeaderConfig } from './base';

export const HEADER_NONE_CONFIG: HeaderConfig = {
    headerType: HeaderType.NONE,
    name: 'None',
    startBytes: [],
    numStartBytes: 0,
    encodesPayloadType: false,
    payloadTypeByteIndex: -1,
    description: 'No start bytes - payload begins immediately (requires external sync)'
};

File: src/struct_frame/boilerplate/py/frame_headers/header_none.py
# Header None - No start bytes
# Relies on external synchronization

from .base import HeaderType, HeaderConfig

HEADER_NONE_CONFIG = HeaderConfig(
    header_type=HeaderType.NONE,
    name="None",
    start_bytes=[],
    num_start_bytes=0,
    encodes_payload_type=False,
    payload_type_byte_index=-1,
    description="No start bytes - relies on external synchronization"
)
